test/cloudflare-handler.test.ts

```diff
@@ -145,7 +141,10 @@ describe("createCloudflareHandler(app)", () => {
 
   it("GET /api/github/oauth/token", async () => {
     const appMock = {
-      checkToken: jest.fn().mockResolvedValue({ id: 1 }),
+      checkToken: jest.fn().mockResolvedValue({
+        foo: "bar",
```

Comment:
I'd prefer not to add made up properties, it might be confusing to contributors and our future selfs. Can we set `data: { id: 1 }` instead of `foo: "bar"`?
```suggestion
        data: { id: 1 },
```

Reply:
could you do the same where you set `foo: "bar"`?

I can do it for you if you are busy, I really appreciate you taking on this important bug fix

Reply:
Done.